01_fit_models.py

```diff
@@ -1,14 +1,18 @@
 import argparse
+import itertools
 import os
 import pickle as pkl
 import time
 import warnings
 from collections import defaultdict
 from os.path import join as oj
-from typing import Callable, List, Tuple
+from typing import Callable, List, Tuple, Set
 
 import numpy as np
 import pandas as pd
+# from bartpy import BART
+from imodels.util.tree_interaction_utils import (get_gt, interaction_fpr, interaction_f1,
```

Comment:
@csinva I think once these are included the tests will pass

Reply:
Okay just included it let's se...